### PR TITLE
Live filtrering bak feature-toggle, noe UU, metrikker og oppdaterte ingresser for testmiljø

### PR DESCRIPTION
**Pull request by taniaholst**
Feature-toggle: veilarbportefolje.live_filtrering

### CODE REVIEW

File: src/components/toolbar/paginering/paginering.tsx [hunk 1]
```diff
@@ -81,23 +85,12 @@ function Paginering(props: PagineringProps) {
...
-        sideStorrelse: selectSideStorrelse(state),
-        seAlle: selectSeAlle(state)
-    };
-};
-
-const mapDispatchToProps = (dispatch, props: OwnProps) => ({
-    endrePaginering: (side, seAlle) => dispatch(pagineringSetup({side, seAlle}))
-});
-
-export default connect(mapStateToProps, mapDispatchToProps)(Paginering);
```
Comment (dragutin-nav) [inline src/components/toolbar/paginering/paginering.tsx:103]: Good that you refactored this, it was difficult to debug and understand logic with the previous logic

File: src/filtrering/filtrering-filter/filterform/aktivitet-filterform.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                            </label>
+                            <input
+                                id={`aktivitet-${kode}-nei`}
+                                name={kode}
+                                value="NEI"
+                                type="radio"
+                                checked={valgteAktiviteter[kode] === 'NEI'}
+                                className="skjemaelement__input radioknapp"
+                                onChange={() => handleRadioChange(kode, 'NEI')}
+                                key={`NEJ, ${verdi}`}
```
Comment (dragutin-nav) [inline src/filtrering/filtrering-filter/filterform/aktivitet-filterform.tsx:98]: can you pls check Is it _NEJ_ in key correct

Comment (taniaholst) [reply src/filtrering/filtrering-filter/filterform/aktivitet-filterform.tsx]: I changed it to NEI now, I think it still works?

File: src/filtrering/filtrering-filter/filterform/double-checkbox-filterform.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+            <div className={'filterform__under-valg'}>
+                <NullstillValgKnapp
+                    dataTestId="double-checkbox-filterform"
+                    nullstillValg={nullstillValg}
+                    form={'utdanning-godkjent-og-bestatt'}
+                    disabled={!(checkBoxValgCol1.length > 0 || checkBoxValgCol2.length > 0)}
+                />
+                {(!harValgCol1 || !harValgCol2) && (
+                    <AlertStripe type="info" className="checkbox-filterform__alertstripe">
+                        {emptyCheckboxFilterFormMessage || 'Ingen veiledere funnet'}
```
Comment (dragutin-nav) [inline src/filtrering/filtrering-filter/filterform/double-checkbox-filterform.tsx:115]: I am not sure that default value should be: 'ingen veiledere funnet', for this file it doesnt seems its related to veiledere..

File: src/filtrering/filtrering-filter/filterform/aktivitet-filterform.tsx (hunk 2 above)
Comment (dragutin-nav) [reply src/filtrering/filtrering-filter/filterform/aktivitet-filterform.tsx]: Yep, I am not sure is it used at all....it still works.

**Review by dragutin-nav — APPROVED** [review]

File: src/filtrering/filtrering-filter/filterform/double-checkbox-filterform.tsx (hunk 3 above)
Comment (taniaholst) [reply src/filtrering/filtrering-filter/filterform/double-checkbox-filterform.tsx:115]: I'll look into it!

**Pull request merged**